_includes/person.html

```diff
@@ -1,8 +1,39 @@
-<div class="card m-2" style="width: 12rem;">
-    <img src="{{include.member.img | default: '/assets/img/team/placeholder.png'}}" style="width: 12rem;" class="card-img-top" alt="DP">
-    <div class="card-body">
-        <h5 class="card-title">{{include.member.name}}</h5>
-        <h6 class="card-subtitle mb-2 text-muted">{{include.member.title}}</h6>
-        <p class="card-text">{{include.member.interests}}</p>
-    </div>
-</div>
\ No newline at end of file
+<link rel="shortcut icon" type="image/x-icon" href="img/icon.ico" />
+<meta charset="utf-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<meta name="description" content="" />
+<meta name="author" content="" />
+
+<!-- Bootstrap Core CSS -->
+<link href="css/bootstrap.min.css" rel="stylesheet" />
+
+<!-- Custom CSS -->
+<link href="css/modern-business.css" rel="stylesheet" />
+
+<!-- Custom Fonts -->
+<link
+	href="font-awesome/css/font-awesome.min.css"
+	rel="stylesheet"
+	type="text/css"
+/>
+<link
+	rel="stylesheet"
+	href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+	integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+	crossorigin="anonymous"
+/>
```

Comment:
you can move these into _includes/head.html file (better put them there to make them globally available)